pgml-dashboard/test_table.sql~

```diff
@@ -0,0 +1,9 @@
+
+SELECT id,relation_name,y_column_name,
```

Comment:
I think this file may have been added by mistake?

Reply:
Yep, will get rid of it and make the frontend changes that Lev recommended and push it.